[
    {
        "type": "message",
        "user": "U1H7F590E",
        "text": "Restore is now always attempted on restart. This being said if a transaction family is added to the config and the validator tries to restore on old data files the above error occurs. To fix this you need to delete the data files created by the previous run. I believe theses files are located in validator\/data. The tutorial is out of date with these changes.",
        "edited": {
            "user": "U1H7F590E",
            "ts": "1477318342.000000"
        },
        "ts": "1477318328.000281"
    },
    {
        "user": "U2TB1TLUE",
        "text": "<@U2TB1TLUE|taylant> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1477319098.000283"
    },
    {
        "type": "message",
        "user": "U2TB1TLUE",
        "text": "i got the same error <@U1VT657T4> , works after deletion of the data.",
        "ts": "1477319329.000284"
    },
    {
        "type": "message",
        "user": "U2TB1TLUE",
        "text": "you have to skip the line in the tutorial `.\/bin\/txnvalidator -v --genesis -F ledger.transaction.integer_key`",
        "edited": {
            "user": "U2TB1TLUE",
            "ts": "1477319909.000000"
        },
        "ts": "1477319594.000285"
    },
    {
        "type": "message",
        "user": "U1V9UML9E",
        "text": "<@U2TB1TLUE>  I will review the tutorial. We updated some of the tutorials for this change, but maybe I missed something!",
        "ts": "1477320117.000287"
    },
    {
        "type": "message",
        "user": "U1VT657T4",
        "text": "<@U1H7F590E> <@U2TB1TLUE> Thanks :slightly_smiling_face:",
        "ts": "1477324015.000288"
    },
    {
        "type": "message",
        "user": "U0YQQRNU8",
        "text": "we will eventually have a 'sawtooth admin clean --blockchain' command to help remove the data, and plan to deprecate '--genesis' flag in favor of a 'sawtooth admin poet-genesis' command to generate the genesis block",
        "ts": "1477342353.000289"
    },
    {
        "type": "message",
        "user": "U0YQQRNU8",
        "text": "the vagrant environment will now work with the 'ubuntu\/xenial64' box file (ubuntu 16.04)",
        "ts": "1477342458.000290"
    },
    {
        "type": "message",
        "user": "U0YQQRNU8",
        "text": "I'm interested in adding centos 7 support, if someone wants to take that on let me know.",
        "ts": "1477342549.000291"
    }
]